cluster/gce/gci/configure.sh

```diff
@@ -546,6 +548,43 @@ function install-containerd-ubuntu {
   sudo systemctl start containerd
 }
 
+function install-auth-provider-gcp {
+  local -r auth_provider_tar="auth-provider-gcp-${DEFAULT_AUTH_PROVIDER_GCP_VERSION}-${HOST_PLATFORM}_amd64.tar.gz"
```

Comment:
Instead of hardcoding `_amd64.tar.gz`, I would use the use `${HOST_ARCH}` (so we can expand it to other arch later if needed).

i.e. follow same naming scheme as NPD:

```
local -r npd_tar="node-problem-detector-${npd_version}-${HOST_PLATFORM}_${HOST_ARCH}.tar.gz"
```

https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure.sh#L290